Changelog.md

```diff
@@ -3,6 +3,7 @@
 ### Changed
 
 -   Updated `nrfutil device` to v2.6.4.
+-   Extracted dataConstrutors to dropdown helper in shared.
```

Comment:
Given that this is not a customer facing change we do not need to put it in the change log